dtfs-central-api/src/v1/controllers/tfm/amendments/tfm-get-amendments.controller.js

```diff
@@ -248,38 +249,39 @@ const findLatestCompletedValueAmendmentByFacilityId = async (facilityId) => {
     try {
       const collection = await db.getCollection('tfm-facilities');
       const amendment = await collection
-        .aggregate([ // TODO SR-8
-          { $match: { _id: ObjectId(facilityId) } },
+        .aggregate([
+          { $match: { _id: { $eq: ObjectId(facilityId) } } },
           { $unwind: '$amendments' },
           {
             $match: {
               $or: [
                 {
-                  'amendments.status': COMPLETED,
-                  'amendments.submittedByPim': true,
-                  'amendments.requireUkefApproval': false,
-                  'amendments.changeFacilityValue': true,
+                  'amendments.status': { $eq: COMPLETED },
+                  'amendments.submittedByPim': { $eq: true },
+                  'amendments.requireUkefApproval': { $eq: false },
+                  'amendments.changeFacilityValue': { $eq: true },
                 },
                 {
-                  'amendments.status': COMPLETED,
-                  'amendments.bankDecision.decision': PROCEED,
-                  'amendments.bankDecision.submitted': true,
-                  'amendments.ukefDecision.value': APPROVED_WITH_CONDITIONS,
-                  'amendments.changeFacilityValue': true,
+                  'amendments.status': { $eq: COMPLETED },
+                  'amendments.bankDecision.decision': { $eq: PROCEED },
+                  'amendments.bankDecision.submitted': { $eq: true },
+                  'amendments.ukefDecision.value': { $eq: APPROVED_WITH_CONDITIONS },
+                  'amendments.changeFacilityValue': { $eq: true },
                 },
                 {
-                  'amendments.status': COMPLETED,
-                  'amendments.bankDecision.decision': PROCEED,
-                  'amendments.bankDecision.submitted': true,
-                  'amendments.ukefDecision.value': APPROVED_WITHOUT_CONDITIONS,
-                  'amendments.changeFacilityValue': true,
+                  'amendments.status': { $eq: COMPLETED },
+                  'amendments.bankDecision.decision': { $eq: PROCEED },
+                  'amendments.bankDecision.submitted': { $eq: true },
+                  'amendments.ukefDecision.value': { $eq: APPROVED_WITHOUT_CONDITIONS },
+                  'amendments.changeFacilityValue': { $eq: true },
                 },
               ],
             },
           },
-          { $sort: { 'amendments.updatedAt': -1, 'amendments.version': -1 } },
-          { $project: { _id: 0, amendments: 1 } },
-          { $limit: 1 },
+          { $sort: { 'amendments.updatedAt': -1, 'amendments.version': -1 } }, // TODO SR-8: Sort order -- no $eq expression needed
+          // TODO SR-8 Changed values in project to true or false to better represent the functionality
+          { $project: { _id: false, amendments: true } },
+          { $limit: 1 }, // TODO SR-8: Does not take an expression -- no $eq expression needed
```

Comment:
Remove todo once noted:
```suggestion
          { $sort: { 'amendments.updatedAt': -1, 'amendments.version': -1 } },
          { $project: { _id: false, amendments: true } },
          { $limit: 1 },
```